lib/Fencer/Rules.hs

```diff
@@ -21,18 +27,36 @@ import qualified Data.Yaml as Yaml
 
 import Fencer.Types
 
+data LoadRulesError
+  = ParseException FilePath Yaml.ParseException
+  | IOEx IOException
```

Comment:
Eh, `LoadRulesError` is not used commonly so we can get away with longer names:

```haskell
data LoadRulesError
  = LoadRulesParseError FilePath Yaml.ParseException
  | LoadRulesIOError IOException
```

Reply:
I like your suggestion because it makes it easier to figure out what type a particular error data constructor comes from. Fixed in an upcoming commit.